src/main/java/WorkShop01.kt

```diff
@@ -41,21 +52,29 @@ object WorkShop01 {
     // PS2. Оператор, который определяет остаток от деления - %
     // /
     // Для проверки задания необходимо запустить тесты в классе WorkShop01P5Test
-    fun evenOddNumber(number: Int) {
-        TODO()
+    fun evenOddNumber(number: Int): String {
```

Comment:
так как If может возвращать значение, то код можно сократить таким вот образом
```suggestion
    fun evenOddNumber(number: Int): String  = if (number % 2 == 0) {
                "Чётное"
            } else {
                "Нечётное"
            }
```